jsdojo/RESTAPI_task/javascriptAPI.js

```diff
@@ -0,0 +1,38 @@
+(function() {
+    'use strict';
+    kintone.events.on('app.record.create.show', (event) => {
+        //  get form settings
+        const params = {
+            'app': 41
+        };
+        kintone.api(kintone.api.url('/k/v1/app/form/fields', true), 'GET', params, (forms) => {
+            let newRow;
+            const setRecord = kintone.app.record.get();
+            const action = forms.properties.table.fields.action_5.options;
+            for (const actions in action) {
```

Comment:
It would be more intuitive for me if the names of the variables at L11 and L12 were the other way around. 
What I'm saying is that the `proposed` code makes more sense than the `current` code, though it's up to you if you rename the variables.

  - current
    ```js
                const action = forms.properties.table.fields.action_5.options;
                for (const actions in action) {
    ```

  - proposed
    ```js
                const actions = forms.properties.table.fields.action_5.options;
                for (const action in actions) {
    ```